src/decompression.jl

```diff
@@ -92,10 +92,11 @@ function TranscodingStreams.process(codec::ZstdDecompressor, input::Memory, outp
 end
 
 function TranscodingStreams.expectedsize(codec::ZstdDecompressor, input::Memory)
-    ret = find_decompressed_size(input.ptr, input.size)
+    ret = LibZstd.ZSTD_getFrameContentSize(input.ptr, input.size)
```

Comment:
This is not a proper replacement. `getFrameContentSize` only reads the size of a single frame.

> Note 5: ZSTD_findDecompressedSize handles multiple frames, and so it must traverse the input to read each contained frame header.  This is fast as most of the data is skipped, however it does mean that all frame data must be present and valid.